library/alloc/src/vec.rs

```diff
@@ -1088,6 +1088,39 @@ impl<T> Vec<T> {
         }
     }
```

Comment:
Could you implement `remove` in terms of `try_remove` to avoid duplicating the code?

Reply:
Is this what you were hinting at: https://github.com/rust-lang/rust/pull/77480/commits/50bb1fda2541ee6ebc932ae198f77a4d8576c8b8#diff-7f954bc61e7a4ce8031361a6f5a6cb71R1068-R1071